### PR TITLE
add 'why is this so awesome' section, fix contributor list

### PR DESCRIPTION
**Pull request by jancborchardt**
Please review @DeepDiver1975 @Gomez @wurstchristoph @zinks- everyone. :)

### CODE REVIEW

**Comment by wurstchristoph** [comment]
nice :+1:

File: README.md [hunk 1]
```diff
@@ -13,29 +12,42 @@ An email app for [ownCloud](https://owncloud.org) (minimum version 8). At the mo
...
-- [Christoph Wurst](https://github.com/wurstchristoph)
-- [Thomas Imbreckx](https://github.com/zinks-)
-- [Plato Leung](https://github.com/PoPoutdoor)
-- [Steffen Lindner](https://github.com/Gomez)
-- past contributors: [Jakob Sack](https://github.com/jakobsack), [Bart Visscher](https://github.com/bartv2), [Sebastian Schmid](https://github.com/sebastian-schmid)
+## Why is this so awesome?
 
-If you’d like to join, just go through the [issue list](https://github.com/owncloud/mail/issues) and fix some. :)
+* :rocket: **Integration with other ownCloud apps!** Currently Contacts & Files – more to come.
+* :+1: **You can use multiple mail accounts!** Unlike [Roundcube](https://roundcube.net) for example.
```
Comment (Xenopathic) [inline README.md]: Perhaps let's avoid naming and shaming :smile: Best would be a straightforward 'Gmail and personal account? No problem!', but if you must, then 'unlike some other webmail solutions'

**Comment by DeepDiver1975** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (jancborchardt) [inline README.md]: Yeah, was also not sure about that. Will change it.

**Comment by jancborchardt** [comment]
The readme is not covered by tests -> merge.

**Pull request merged**